lapce-data/src/data.rs

```diff
@@ -227,13 +252,46 @@ impl LapceData {
             windows.insert(window.window_id, window);
         }
 
-        if let Some((window_id, _)) = windows.iter().next() {
+        if let Some((window_id, data)) = windows.iter().next() {
             for file in files {
-                let _ = event_sink.submit_command(
-                    LAPCE_UI_COMMAND,
-                    LapceUICommand::OpenFile(file.to_path_buf(), false),
-                    Target::Window(*window_id),
-                );
+                let file_path = match file.path.canonicalize() {
+                    Ok(v) => v,
+                    _ => pwd.join(file.path),
+                };
+                for (widget_id, _) in &data.tabs {
+                    if let Some(pos) = file.linecol {
+                        // jump to line and column
+                        if let Err(err) = event_sink.submit_command(
+                            LAPCE_UI_COMMAND,
+                            LapceUICommand::JumpToLineColLocation(
+                                None,
+                                EditorLocation {
+                                    path: file_path.clone(),
+                                    position: Some(pos), // line info is included in column variable
+                                    scroll_offset: None,
+                                    history: None,
+                                },
```

Comment:
There is very little time, so I did not dig deep. But there seems to be a bug here. The program crashes if you try to access ".../lapce/README.md:5" or ".../lapce/README.md:5:1". It looks like the problem is UTF-8 encoding and different length of characters in bytes.

Reply:
This bug seems to be present in all variations of this (that #1886?) pull request :-).

Reply:
so are we gonna solve the bug or what?

Reply:
This issue is not introduced in this PR (but it can be manifested because of this PR) so I'll leave it for someone else to figure that out